tsconfig.json

```
@@ -6,7 +6,10 @@
     "target": "ES2020",
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "jsx": "react-jsx",
-
+    "moduleResolution": "bundler",     // keep this for Vite
```

Biome (2.1.2)

[error] The key moduleResolution was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)